src/Tribe/Front_Page_View.php

```diff
@@ -3,11 +3,29 @@
  * Provides an option to position the main events view on the site homepage.
  */
 class Tribe__Events__Front_Page_View {
+
+
+	/**
+	 * The ID used to identify the virtual page, using a -10 for no particular reason, but avoding -1 as is regular
+	 * used as infinite or any other popular reference.
+	 *
+	 * @since TBD
+	 *
+	 * @var int
+	 */
+	private $HOME_VIRTUAL_ID = -10;
```

Comment:
Lets not use uppercase variable, that is reserved for Constants. Just use a normal prop.